charts/openstack-cluster/values.yaml

```diff
@@ -73,7 +73,8 @@ kubeNetwork:
 # Settings for the OpenStack networking for the cluster
 clusterNetworking:
   # Custom nameservers to use for the hosts
-  dnsNameservers:
+  # Deprecated; use `clusterNetworking.internalNetwork.dnsNameservers` instead.
+  dnsNameservers: []
```

Comment:
I think just remove this completely

Reply:
Perhaps? It would keep the values file cleaner, but I note that the `magnum-capi-helm` driver still uses this - I figured it would be nicer to keep it documented? (Commenting out the actual field may help?)

This appears to be the practice [in other helm charts](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L4232), but happy to change if the code style here differs for a reason. I'm just interested in understanding more.

Reply:
Maybe commented out is good.

Reply:
Done